wikimedia/wikipedia-ios
# Add profile icon to Settings (when in tab bar)

**Phabricator:** https://phabricator.wikimedia.org/T374879

### Notes
This PR displays the profile icon on the Settings view, when the Explore feed is disabled.

### Test Steps
1. Go to branch 7.5.9. Fresh install, launch app, then go to Settings > Explore preferences > Turn off Explore tab toggle. App should switch so that the Settings view is embedded in the first tab.
2. Switch to this PR branch. Confirm you see profile icon on Settings tab. Tap icon, confirm profile view presents.
